app/scripts/services/resourceAlerts.js

```diff
@@ -18,6 +18,18 @@ angular.module("openshiftConsole")
       }
 
       var alerts = {};
+      // Handle evicted pod warnings specially so that evicted pods don't appear in the donut.
+      if (_.some(pods, { status: { phase: 'Failed', reason: 'Evicted' } })) {
+        alerts['pod_evicted'] = {
+          type: 'warning',
+          message: 'Pods have been evicted.',
```

Comment:
feel like the message could be a bit more helpful than that, @derekwaynecarr said pods are typically evicted because they are using more resources than they requested